helm_deploy/values-dev.yaml

```diff
@@ -40,4 +40,4 @@ generic-service:
   allowlist: null
 
 generic-prometheus-alerts:
```

Comment:
```suggestion
generic-prometheus-alerts:
  businessHoursOnly: true
```

Reply:
I'd suggest adding this as service shutdown overnight on non-prod environments.

Reply:
OK thanks, done